### PR TITLE
[irods/irods#6330] Added documentation for unattended installs (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Comment by trel** [comment]
content looks good.   seems our headers are off-by-one.   can fix that separately.

might want to move this section 'up' a level, and perhaps just above/below 'non-package-install'.

File: docs/getting_started/installation.md [hunk 1]
```diff
@@ -180,6 +180,337 @@ schema_name - irods_environment
...
+- [https://github.com/irods/irods/blob/main/schemas/configuration/v4/unattended_installation.json.in](https://github.com/irods/irods/blob/main/schemas/configuration/v4/unattended_installation.json.in)
+
+This form of installation is known as an **Unattended Install**. Using this form of setup can help with automated deployments.
+
+To perform an **Unattended Install**, you'd run the following:
+```bash
+$ (sudo) python3 /var/lib/irods/scripts/setup_irods.py --json_configuration_file /path/to/your/json/input/file
+```
+
+Below is an example of what the input file for an iRODS Catalog Provider might contain:
```
Comment (trel) [inline docs/getting_started/installation.md:196]: could make a `### Catalog Service Provider` here...

and a `### Catalog Service Consumer` down below... could make the nav a bit easier to see.

**Comment by trel** [comment]
i like it - please squash em.

**Comment by korydraughn** [comment]
Added #

**Pull request merged**